### DISCUSSION TITLE
Wondering regarding Andronix

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#10258] [General]
Just wondering how many people here are aware of the app Andronix. 
It is truly awesome.
It enables you to run a full-blown Linux implementation within Termux.

Explanation:- A Linux implementation via the Andronix app uses proot within Termux to run a full-blown Linux within Termux. It's wonderful! Once you have it up and running, the actual Andronix app can be uninstalled from your phone as it is only used for the initial system setup.
I have installed both Ubuntu and Debian within Termux but finally settled on Debian buster as well supported and quite small.

**Comment by macerg** [comment]
[Andronix on Google play ](https://play.google.com/store/apps/details?id=studio.com.techriz.andronix&gl=US)

**Reply by MrAdityaAlok** [reply]
Maybe you do not know, we too have had our own [proot distros](https://github.com/termux/proot-distro). Why do you need a third party one (unless you pay for there modded distro's)? You can even add your own distros too, with this utility.

*This is suggest a suggestion/advice. I am not questioning your choice 😅*

**Reply by macerg** [reply]
No, wasn't aware of termux proot distros, thanks, I'll look into them. Yes, I payed a vsmall amount and have a modded Debian buster system. Thanks again.

**Reply by macerg** [reply]
Hello MrAdityaAlok. I note that the distros are 'bare bones' systems. Do they come with desktop x11 environments? If not, are there instructions on how to install?

**Reply by MrAdityaAlok** [reply]
> Hello MrAdityaAlok. I note that the distros are 'bare bones' systems. Do they come with desktop x11 environments?
> 

No, they do not come with x11.

 > If not, are there instructions on how to install? 

You may search online. Standard way of installing x11 with vnc for respective distro should work.
